_posts/2022-02-15-a-quick-opensearch-primer.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "A Quick OpenSearch Primer"
+authors:
+- nateboot
+
+date: 2022-02-15
+categories:
+ - intro
+
+excerpt: "OpenSearch is an open source project built on top of Apache Lucene, a powerful indexing and search library.  Even if you’ve never heard of an indexing library before, you’re still probably more familiar with indexing than you think. "
+---
+
+### First time here?
+
+Have you only recently heard of OpenSearch and are curious just what it is? Here’s some help getting on your feet. First, about indexing and full text search as a core function. Also, how OpenSearch extends those things into something more than a search library. OpenSearch provides a basis for many useful workflows meant to analyze, visualize, and aggregate data. 
+
+### So what is OpenSearch then? 
+
+OpenSearch is an open source project built on top of Apache Lucene, a powerful indexing and search library.  Even if you’ve never heard of an indexing library before, you’re still probably more familiar with indexing than you think. The quantity of data businesses maintain in this age necessitates it.  Apache Lucene provides OpenSearch the ability to maintain large volumes of full documents, arbitrary text as well as non-textual data while maintaining several indices to make searching through each of them efficient.  
```

Comment:
open-source is preferred before a noun

I suggest acknowledging its origins, open-source fork of Elasticsearch and Kibana

I would explain that indexing is like preparing data to make searching fast (rather than efficient).

Reply:
Seems reasonable to me. They're a big part of its history. No reason why I shouldn't acknowledge their existence.

Reply:
- resolved open source hyphenation